django/contrib/postgres/functions.py

```diff
@@ -3,17 +3,9 @@
 
 class RandomUUID(Func):
     template = 'GEN_RANDOM_UUID()'
-
-    def __init__(self, output_field=None, **extra):
```

Comment:
@sir-sigurd are you saying `RandomUUID(1, output_field=None)` uses to raise a `TypeError` and it doesn't anymore? I don't think we should refrain from breaking the use of positional `output_field` (e.g. `RandomUUID(UUIDField)`).